src/moonlink/src/table_notify.rs

```diff
@@ -69,25 +69,35 @@ pub enum TableEvent {
         xact_id: Option<u32>,
         lsn: u64,
         is_copied: bool,
+        is_recovery: bool,
```

Comment:
do you think it's better to make it a enum?
for example
```rs
enum RowWriteState {
 Recovery,
 InitialCopy,
 Normal,
}
```
My concern is we could have more and more special states, and would to reduce the boolean states